client/web/src/hooks/node-data.ts

```diff
@@ -36,7 +36,16 @@ export default function useNodeData() {
   const [isPosting, setIsPosting] = useState<boolean>(false)
 
   const fetchNodeData = useCallback(() => {
-    apiFetch("api/data")
+    const urlParams = new URLSearchParams(window.location.search)
+    const nextParams = new URLSearchParams()
+    const token = urlParams.get("SynoToken")
+    if (token) {
+      nextParams.set("SynoToken", token)
+    }
+    const search = nextParams.toString()
+    const url = `api/data${search ? `?${search}` : ""}`
```

Comment:
This kind-of mirrors the logic down in the post function below. Could share some of it. But that can be cleaned up in a follow-up, I'm happy to do that with the localapi proxy work.

Reply:
that was what I was referring to in my comment at the top of the PR 😛

Reply:
Oh whoops, I didn't process that! Okay sgtm!